update.sh

```diff
@@ -33,23 +33,24 @@ hour=$(echo $hour | sed 's/^0*//')
 # Different desktop environment implementations
 case $XDG_CURRENT_DESKTOP in
 	Mint|Mate) setcmd="gsettings set org.mate.background picture-uri";;
-	Cinnamon) setcmd="gsettings set org.cinnamon.background picture-uri";;
-	*) setcmd="gsettings set org.gnome.desktop.background picture-uri";; # GNOME/Unity, default
+	MATE) setcmd="gsettings set org.mate.background picture-filename ";;
+	Cinnamon) setcmd="gsettings set org.cinnamon.background picture-uri file://";;
+	*) setcmd="gsettings set org.gnome.desktop.background picture-uri file://";; # GNOME/Unity, default
```

Comment:
I'm using macOS now, before that (when I made this repo) I used Ubuntu. Sounds like you're right about the caps/no caps thing and I think we should be fine with that knowledge, I just wanted to be sure